app/controllers/mixins/containers_common_mixin.rb

```diff
@@ -79,6 +81,20 @@ def show_container(record, controller_name, display_name)
                       :url  => "/#{controller_name}/show/#{record.id}" \
                                "?display=#{@display}&refresh=n")
       perf_gen_init_options # Intialize options, charts are generated async
+    elsif @display == "compliance_history"
+      count = params[:count] ? params[:count].to_i : 10
+      session[:ch_tree] = compliance_history_tree(record, count).to_json
+      session[:tree_name] = "ch_tree"
+      session[:squash_open] = (count == 1)
+      if count == 1
+        drop_breadcrumb(:name => _("%{name} (Latest Compliance Check)") % {:name => record.name},
+                        :url  => "/#{controller_name}/show/#{record.id}?display=#{@display}&refresh=n")
+      else
+        drop_breadcrumb(:name => _("%{name} (Compliance History - Last %{number} Checks)") %
```

Comment:
this method is already huge, is number defined here?  I can't tell if it's a method or local variable.